File: python/blink.py
from time import sleep
import RPi.GPIO as GPIO #gives us access to the GPIO pins
GPIO.setmode(GPIO.BCM)  #sets the pin numbering system we want to use

LED = 17                #pin number driving the LED
delay = 1               #change this number to adjust the frequency at which the light blinks
GPIO.setup(LED,GPIO.OUT)

while True:
    GPIO.output(LED,True)  #turn LED on
    sleep(delay)

    GPIO.output(LED,False) #turn LED off
    sleep(delay)

File: wiringPi/blink/blink.c
#include <wiringPi.h>
#include <stdio.h>

const int LED = 0;
const int CHANGE_DELAY = 500;

void setup()
{
	pinMode(LED, OUTPUT);
	digitalWrite(LED, 0);
}

int main(void)
{
	if(wiringPiSetup() == -1)
		return 1;
	
	while(1)
	{
		digitalWrite(LED, 1);
		delay(CHANGE_DELAY);

		digitalWrite(LED, 0);
		delay(CHANGE_DELAY);
	}
	return 0;
}
